src/Core/Components/List/FluentPersona.razor.cs

```diff
@@ -49,6 +49,12 @@ public partial class FluentPersona : FluentComponentBase
     [Parameter]
     public string? ImageSize { get; set; }
 
+    /// <summary>
+    /// Gets or sets the position of the image.
+    /// </summary>
+    [Parameter]
+    public PersonaImagePosition? ImagePosition { get; set; }
```

Comment:
I think it would be more logical to follow the "same philosophy" used by React:
by calling the new attribute `TextPosition`.
 
https://react.fluentui.dev/?path=/docs/components-persona--default#text-alignment